starcheck/pcad_att_check.py

```diff
@@ -67,13 +73,17 @@ def make_pcad_attitude_check_report(backstop_file, or_list_file=None, mm_file=No
     lines = []  # output report lines
 
     mm = read_maneuver_summary(mm_file)
-    q = [mm[0][key] for key in ['q1_0', 'q2_0', 'q3_0', 'q4_0']]
+    q = Quaternion.normalize([mm[0][key] for key in ['q1_0', 'q2_0', 'q3_0', 'q4_0']])
     bs = read_backstop(backstop_file)
     simfa_time, simfa = recent_sim_history(DateTime(bs[0]['date']).secs,
                                            simfocus_file)
     simpos_time, simpos = recent_sim_history(DateTime(bs[0]['date']).secs,
                                              simtrans_file)
-    initial_state = {'q_att': q,
+
+    initial_state = {'q1': q[0],
```

Comment:
This is starting to get to a point that keeps hopper happy, but this is using the maneuver summary to set initial quats... which is not consistent with replan/reopen when the backstop passed to hopper starts before the DOT and maneuver summary.  And then we've got all of the back and forth about checking the pre-DOT interval.

Mostly I think we've been spared more errors from hopper here because the rest of the stuff that hopper is trying to do is not implemented (but it looks to be configured to fetch stars and try to build catalogs to check even though I don't think we are using any of those features).  Might be easiest just to comment out everything after the attitude checking and come back to hopper improvements.

Reply:
Updated to set initial quats from ATTITUDE history file.